common/tools/dev-tool/src/commands/customization/init.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import path from "path";
+
+import { resolveProject } from "../../util/resolveProject";
+import { createPrinter } from "../../util/printer";
+import { leafCommand } from "../../framework/command";
+import { makeCommandInfo } from "../../framework/command";
+
+import { ensureDir, copy, readdir, stat as fsStat, rm, unlink } from "fs-extra";
```

Comment:
Did you consider importing from `fs/promises`?

Reply:
+1 for `fs/promises`, I haven't needed `fs-extra` recently